script.js

```diff
@@ -157,8 +157,33 @@ const portfoliopopup = (value) => {
   </div>`;
   document.body.appendChild(popcontainer);
   const closingpop = document.querySelector('#popclosingbutton');
-  closingpop.addEventListener('click', () => document.body.removeChild(document.querySelector('.popupcontainer')));
+  closingpop.addEventListener('click', () => document.body.removeChild(document.querySelector('.popupcontainermain')));
 };
 
 const buttonportfolio = document.querySelectorAll('.orangebottom');
-buttonportfolio.forEach((button) => button.addEventListener('click', portfoliopopup));
\ No newline at end of file
+buttonportfolio.forEach((button) => button.addEventListener('click', portfoliopopup));
+
+// *FORM VALIDATION:
+
+const formemail = document.getElementById('inputemail');
+const form = document.getElementById('contactform');
+const labelerror = document.getElementById('errormessage');
+const labelcheck = document.getElementById('checkmessage');
+
+form.addEventListener('submit', (e) => {
+  const mailformat = (/^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/);
+  const nonlowercaseletters = /[^a-z]/g;
+  if (formemail.value.match(nonlowercaseletters)) {
+    if (formemail.value.match(mailformat)) {
+      labelcheck.classList.remove('nodisplay');
+      labelerror.classList.add('nodisplay');
+    } else {
+      labelerror.classList.remove('nodisplay');
+      labelcheck.classList.add('nodisplay');
+    }
+  } else {
+    labelerror.classList.remove('nodisplay');
+    labelcheck.classList.add('nodisplay');
+  }
+  e.preventDefault();
+});
```

Comment:
- [ ] You should check if the email is all lowercase by comparing if the email and it's lowercase converted version are the same,` ie: email === email.toLowerCase(),`
- Make sure after the validation is successful, you submit the form successfully 
![image](https://user-images.githubusercontent.com/43877689/180446295-a8523c72-4450-4250-8787-1adeca959316.png)